views/login.handlebars

```diff
@@ -0,0 +1,21 @@
+    <section class="p-2">
+        <div class="card text-center">
+            <div class="card-header">
+              Log In
+            </div>
+            <div class="card-body">
+                <form>
+                    <div class="form-group p-2">
+                      <label for="inputEmail">Email address</label>
```

Comment:
`for` attribute needs to be `'email'`, along with the `id` attribute for `<input>` needing to be `'email'` on line 10 in order to link up properly with api routes

Reply:
Password elements' respective `id` and `for` attributes should just be password in order to link up with api routes on lines 13 and 14

Reply:
You can disregard the above, I believe all that needs to be added is a `name` attribute for each `<input>` element. The names for each input should be `'email'` and `'password'`. I don't believe the other attributes matter as far as `req.body` goes.